components/shared/Button/button.tsx

```diff
@@ -8,10 +8,25 @@ export interface ButtonsProps extends React.ButtonHTMLAttributes<HTMLButtonEleme
   rel?: string;
   target?: string;
   showLoadingText?: boolean;
+  loadingText?: string;
 }
 
 const Button = React.forwardRef<HTMLElement, ButtonsProps>(
-  ({ className, children, loading, disabled, variant, showLoadingText = true, onClick, href, ...props }, ref) => {
+  (
+    {
+      className,
+      children,
+      loading,
+      disabled,
+      variant,
+      showLoadingText = true,
+      loadingText = "Loading...",
```

Comment:
The default loading text is `"Loading..."` to not break it elsewhere in the app.